pkg/scheduler/scheduler.go

```diff
@@ -255,6 +257,14 @@ func (s *Scheduler) Run(ctx context.Context) {
 	s.informerFactory.Start(stopCh)
 	s.informerFactory.WaitForCacheSync(stopCh)
 
+	s.clusterWorker = util.NewAsyncWorker(util.Options{
+		Name:               "scheduler cluster watcher",
+		KeyFunc:            func(obj interface{}) (util.QueueKey, error) { return obj, nil },
```

Comment:
Why enqueue the whole cluster obj? Why not the namespace key?

Reply:
If the key is converted, only the new cluster object can be obtained during reconcile processing. As a result, some events will be lost.